src/api/task_info.yaml

```diff
@@ -0,0 +1,72 @@
+name: A unique identifier. Can only contain lowercase letters, numbers or underscores.
```

Comment:
I wouldn't do this now, but I think we should try to move as many of these fields (if not all) to `_viash.yaml`.